### ISSUE TITLE
US Customary Units

### ISSUE THREAD

**Issue by GregJohnStewart** [unitsofmeasurement/indriya#357]
Hello,

It would be great to have the US customary units available in `Units` (or similar listing of standardized units).

While I agree Metric is of course better, there is enough of a market to warrant the inclusion of these units. It shouldn't be too hard, as they are even based on their Metric counterparts by the US.

https://en.wikipedia.org/wiki/United_States_customary_units

**Comment by keilw** [comment]
Hello Greg,

Thanks, but they already are in [systems-common](https://github.com/unitsofmeasurement/uom-systems/tree/master/common) as an extension module on top of Indriya. Please check it out and if you're missing a unit there or find a problem file a ticket in its [issues](https://github.com/unitsofmeasurement/uom-systems/issues).

**Comment by GregJohnStewart** [comment]
Thanks!

... do you guys need help with documentation? Just getting started with the library, and it looks like proper docs are incredibly sparse

**Comment by keilw** [comment]
Yes happy to take help, especially [uom-systems](https://github.com/unitsofmeasurement/uom-systems) had contributions here and there before, see files like `CONTRIBUTORS` or `contributors.txt` in the subsequent modules, and it's a little easier there because unlike this module (the RI of JSR 385) you do not need to join the JCP as an Associate Member or higher.
